flashinfer_bench/bench/evaluators/sampling.py

```
@@ -81,12 +81,15 @@ def build_baseline(
-            ms = time_runnable(ref_runnable, inp, cfg.warmup_runs, cfg.iterations, device)
-            latencies.append(ms)
+        if cfg.profile_baseline:
+            latencies: List[float] = []
+            for inp in inputs:
+                ms = time_runnable(ref_runnable, inp, cfg.warmup_runs, cfg.iterations, device)
+                latencies.append(ms)
 
-        mean_latency_ms = sum(latencies) / float(len(latencies))
+            mean_latency_ms = sum(latencies) / float(len(latencies))
+        else:
+            main_latency_ms = 0.0
```

Ruff (0.14.10)

Local variable `main_latency_ms` is assigned to but never used

Remove assignment to unused variable `main_latency_ms`

(F841)